backend/src/Manager/Image/ImageManager.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Manager\Image;
+
+use App\AutoMapping;
+use App\Entity\ImageEntity;
+use App\Repository\ImageEntityRepository;
+use App\Request\Image\ImageCreateRequest;
+use Doctrine\ORM\EntityManagerInterface;
+
+class ImageManager
+{
+    public function __construct(private AutoMapping $autoMapping, private EntityManagerInterface $entityManager, private ImageEntityRepository $imageEntityRepository)
+    {
+    }
+
+    public function create(ImageCreateRequest $request)
+    {
+        $imageEntity = $this->autoMapping->map(ImageCreateRequest::class, ImageEntity::class, $request);
+
+        $this->entityManager->persist($imageEntity);
```

Comment:
how does this even work?
where entityManager defined? also others?

Reply:
entityManager and other class properties were being defined in the constructor function definition.